test/blackbox-tests/test-cases/pkg/ocamlformat/patch-extra-files.t

```diff
@@ -0,0 +1,71 @@
+This is a current bug, 'dune fmt' misses extra files from "dev-tools.locks" directory for the first run.
```

Comment:
I don't really understand the bug from this description

Reply:
I'll rework it.

Reply:
I think it could be more comprehensible for now.